tasks/testTasks.ts

```diff
@@ -102,12 +102,7 @@ function createOmniSharpTestSubTasks() {
         await runJestTest(jestOmniSharpUnitTestProjectName);
     });
 
-    const omnisharpIntegrationTestProjects = [
-        'singleCsproj',
-        'slnWithCsproj',
-        'slnFilterWithCsproj',
-        'BasicRazorApp2_1',
-    ];
+    const omnisharpIntegrationTestProjects = ['singleCsproj', 'slnWithCsproj', 'slnFilterWithCsproj', 'RazorApp'];
```

Comment:
OmniSharp razor project is still named BasicRazorApp2_1. See https://github.com/dotnet/vscode-csharp/tree/main/test/omnisharp/omnisharpIntegrationTests/testAssets/BasicRazorApp2_1